pkg/driver/mount_test.go

```diff
@@ -64,8 +68,52 @@ func TestMakeFile(t *testing.T) {
 		t.Fatalf("Expect no error but got: %v", err)
 	}
 
+	if mountObj.MakeFile(targetPath) != nil {
+		t.Fatalf("Expect no error but got: %v", err)
+	}
+
 	if exists, err := mountObj.ExistsPath(targetPath); !exists {
 		t.Fatalf("Expect no error but got: %v", err)
 	}
 
 }
+
+func TestExistsPath(t *testing.T) {
```

Comment:
Could you add a happy case for this?

Reply:
The happy case is covered in the above two tests. Do you see a value for a separate one?